api/lib/reducers/polkadotV0-reducers.js

```diff
@@ -449,7 +449,15 @@ function rewardsReducer(network, validators, rewards, reducers) {
   return allRewards
 }
 
-function dbRewardsReducer(validatorsDictionary, dbRewards) {
+function dbRewardsReducer(validatorsDictionary, dbRewards, withHeight) {
+  if (withHeight) {
+    return dbRewards.map((reward) => ({
+      id: `${reward.validator}_${reward.denom}_${reward.height}`,
```

Comment:
I don't get this so well, this `withHeight` logic. I mean, shouldn't it always be the case, that ID goes with height? Why not make it always like this?

Ah, maybe because Cosmos hasn't this logic of reward per height... Oh, I see. OK